code-scanning/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Dependency Review",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Code Scanning",
```

Comment:
You can replaced this with "Dependency Review" which is more appropriate. I have a PR which will allow that to be processed. Will most probably merge that tomorrow my day time.

Reply:
Sorry - did you mean the `name: "Dependency Review"` or the `categories` tag?

In the tag case, I see other two-word tags with both caps (and Dependency Review is the feature, we captialize it in other docs)

In the "name" case, the other YAML file also names this `Dependency Review` - will all those need updates as well if we change this `name`?

basically - what exactly should I update and in which file to match up with your PR change? Also - if it's easier for you to make a push to this PR directly to make the update feel free to do so, as I may not here back before tomorrow (our target ship date!) due to async/ time zones 🙇 

thanks again for the callout!